lib/matplotlib/axes/_axes.py

```diff
@@ -2561,7 +2563,12 @@ def pie(self, x, explode=None, labels=None, colors=None,
         if len(x) != len(explode):
             raise ValueError("'explode' must be of length 'x'")
         if colors is None:
-            colors = ('b', 'g', 'r', 'c', 'm', 'y', 'k', 'w')
+            get_next_color = self._get_patches_for_fill.get_next_color
+        else:
+            color_cycler = itertools.cycle(colors)
```

Comment:
<bikeshed>`color_cycle` instead of `color_cycler` as it is a cycle object not a cycler object (sorry, I am really bad at naming things)</bikeshed>